admin_manual/installation/system_requirements.rst

```diff
@@ -12,31 +12,33 @@ Nextcloud needs a minimum of 128MB RAM, and we recommend a minimum of 512MB.
 Recommended setup for running Nextcloud
 ---------------------------------------
 
-For best performance, stability, support, and full functionality we recommend:
+For best performance, stability, and functionality we recommend:
 
 * Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
 * MySQL/MariaDB
 * PHP 7.0 +
 * Apache 2.4 with mod_php
 
-Supported platforms
--------------------
+For a Nextcloud installation with efficient and reliable scaling over ~100 users, see our `deployment recommendations <https://portal.nextcloud.com/article/nextcloud-deployment-recommendations-7.html>`
```

Comment:
changing"our" with "the", makes it sound less possessive.

Reply:
you rock.